hlc/planner/planner.py

```diff
@@ -0,0 +1,43 @@
+# from hlc.config.constants import *
+from hlc.planner.helper import Position, Pose, HLAction
+
+
+def apply_actions(actions, position, plan):
+    for a in actions:
+        plan.append(a)
+        position.update(a)
```

Comment:
I really like that one. I thought of something like that before, but didn't quite know how to implement it properly.